<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>resume.utf8.md</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Home</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    <span class="fa fa-gear"></span>
     
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-header">p8105 - Data Science</li>
    <li>
      <a href="hw4_dashboard.html">Instacart data visualization</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="resume.html">Resume</a>
</li>
<li>
  <a href="mailto:&lt;elframme@hotmail.com&gt;">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://github.com/Raarbiarsan1899">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://linkedin.com/in/zhou-fang-894274139">
    <span class="fa fa-linkedin fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<center>
<h3>
ZHOU FANG, PH.D.
</h3>
</center>
<center>
<em>100 Haven Ave., 19F, New York, NY, 10032</em>
</center>
<center>
<em>Tel: (216) 801-1399 Email: <a href="mailto:zf2213@cumc.columbia.edu" class="email">zf2213@cumc.columbia.edu</a></em>
</center>
<p><br></p>
<div id="work" class="section level4">
<h4>WORK</h4>
<ul>
<li><p align="text-align: left;">
Postdoctoral Scholar, Genomic and Genetic – Case Western Reserve University <span style="float: right;"> 2015 – 2018 </span>
</p></li>
</ul>
</div>
<div id="education" class="section level4">
<h4>EDUCATION</h4>
<ul>
<li><p align="text-align: left;">
M.S, Biostatistics – Columbia University <span style="float: right;"> 2018 - 2020 (expected) </span>
</p></li>
<li><p align="text-align: left;">
Ph.D, Biology/Genetics – Fudan University <span style="float: right;"> 2008 – 2015 </span>
</p></li>
<li><p align="text-align: left;">
Bachelor of Science, Biological Science – Fudan University <span style="float: right;"> 2004 – 2008 </span>
</p></li>
</ul>
<p><br></p>
</div>
<div id="major-research-fields" class="section level4">
<h4>MAJOR RESEARCH FIELDS</h4>
<ul>
<li><p><strong>Genome-wide CRISPR Screening</strong> [1 and several additional manuscripts under preparation]</p>
<p><em>Short description</em>: Genome-wide CRISPR screening is a powerful tool to investigate gene functions. However, the difficult parts are the unbiased amplification of sgRNA library at plasmid, virus, and cell levels. During my postdoctoral study, I successfully established a genome-wide CRISPR screening platform for a newly formed research team, and applied it on hESC differentiation system and MIN6 cells. Besides, I self-studied R programming and accomplished the data anlysis tasks assigned by the team principle investigator.</p>
<p><em>Roles</em>: genome editing, cell culturing, embryonic stem cell culturing, flow cytometry data analysis (R), virus culturing, library preparation, GO analysis (PANTHER), gene pathway analysis (STRING, Cytoscape)</p></li>
<li><p><strong>Cardiovascular co-morbidities in type II diabetes patients</strong> [2, 6, 7, 8]</p>
<p><em>Short description</em>: According to epidemic studies, half of diabetes patients will develop cardiovascular diseases. However, the prognosis varies from person to person. Molecular markers, SNPs can be powerful in predicting the clinical implications of diabetes patients with heart diseases. I did a stepwise multivariate linear regression to find relation between several serum markers and cardiovascular implications using SPSS. I also participated in a gene function study and an environment-gene interaction study.</p>
<p><em>Roles</em>: data collecting, data cleaning (SPSS), performing stepwise multivariate regression, drawing ROC curve</p></li>
<li><p><strong>Functional study of dysregulated proteins in hepatocellular carcinoma</strong> [3, 4, 5]</p>
<p>Protein expression are dramatically changed in malignant tumor cells. Dysregulation of some essential proteins, such as P53, contributes greatly in the progression of many kinds of cancers. In this project, I assessed several dysregulated proteins on cell growth, apoptosis of liver cancer cell lines in vitro. In particular, I designed, conducted, and published a research paper as first author.</p>
<p><em>Roles</em>: genome editing, virus culturing, bacteria culturing, flow cytometry, DNA transfection, meta-analysis of gene-expression, viral DNA and RNA quantification, in-vivo protein half-life analysis, co-immunoprecipitation</p></li>
</ul>
<p><br></p>
</div>
<div id="skills" class="section level4">
<h4>SKILLS</h4>
<ul>
<li><p><strong>Molecule and cell biology</strong>: 8 years benchwork experience, CRISPR screening, lentivirus production, stem cell culturing, Hiseq-2500 DNA library preparation, Gibson Assembly, Chromosome Conformation Capture (3C), protein purification (Tn5 protein), cell staining and flow cytometry, primer design, and basic skills like PCR, Western blotting.</p></li>
<li><p><strong>Bioinformatics</strong>: Famaliar with UCSC genome browser, GEO database, low throughput DNA alignment (Vector NTI), Pathway analysis (STRING, Cytoscape), GO analysis (Panther), flowCore package (R)</p></li>
<li><p><strong>Biostatistics</strong>: 2 years work experience, data cleaning, stepwise multivariate regression analysis, ROC curve, meta analysis, basic hypothesis testing</p></li>
<li><p><strong>R programming</strong>: 1 year work experience, flow cytometry single-cell data analysis, practice through several courses</p></li>
<li><p><strong>Python</strong>: practice through several courses</p></li>
</ul>
<p><br></p>
</div>
<div id="professional-experience" class="section level4">
<h4>PROFESSIONAL EXPERIENCE</h4>
<ul>
<li><p align="text-align: left;">
Research assistant – Huashan Hospital, Fudan University <span style="float: right;"> 2008 – 2010 </span>
</p></li>
</ul>
</div>
<div id="awards" class="section level4">
<h4>AWARDS</h4>
<ul>
<li><p align="text-align: left;">
3rd Prize Scholarship <span style="float: right;"> 2010 – 2011 </span>
</p></li>
<li><p align="text-align: left;">
2nd Prize Scholarship <span style="float: right;"> 2011 – 2012 </span>
</p></li>
</ul>
<p><br></p>
</div>
<div id="publications" class="section level4">
<h4>PUBLICATIONS</h4>
<ol style="list-style-type: decimal">
<li><p><strong>Zhou Fang</strong>, Chen Weng, Ran Tao, Haiyan Li, Xiaoxiao Liu, Grace Bogos, Yanxin Pei, FulaiJin, Yan Li. Single cell heterogeneity analysis and CRISPR screen identify key β cell-specific disease genes. <em>Cell Reports</em>. <em>Minor revision</em></p></li>
<li><p><strong>Fang Z</strong>, Zhou L, Bao Y, Ding W, Shi H, Luo X, Hu R. Association of NT-proBNP and multiple biomarkers with severity of angiographic coronary artery disease in diabetic and pre-diabetic Chinese patients. <em>PLoS One</em>. 2011;6(8):e22563.</p></li>
<li><p><strong>Fang Z</strong>, Zhou L, Jiang S, Cao L, Yu L. UNC50 prompts G1/S transition and proliferation in HCC by regulation of epidermal growth factor receptor trafficking. <em>PLoS One</em>. 2015 Mar 4;10(3):e0119338.</p></li>
<li><p>Xian L, Zhao J, Wang J, <strong>Fang Z</strong>, Peng B, Wang W, Ji X, Yu L. p53 Promotes proteasome-dependent degradation of oncogenic protein HBx by transcription of MDM2. MolBiol Rep. 2010 Jul;37(6):2935-40.</p></li>
<li><p>Zhao J, Wang C, Wang J, Yang X, Diao N, Li Q, Wang W, Xian L, <strong>Fang Z</strong>, Yu L. E3 ubiquitin ligase Siah-1 facilitates poly-ubiquitylation and proteasomal degradation of the hepatitis B viral X protein. FEBS Lett. 2011 Oct 3;585(19):2943-50.</p></li>
<li><p>Bao YY, <strong>Fang Z</strong>, Zhou LN, Hu RM, Ding W. [Effect of adipose differentiation-related protein on the apoptosis of H9c2 cells by palmitic acid]. Xi Bao Yu Fen ZiMian Yi XueZaZhi. 2011 Apr;27(4):382-5. Chinese.</p></li>
<li><p>Tang ZH, <strong>Fang Z</strong>, Zhou L. Human genetics of diabetic vascular complications. J Genet. 2013 Dec;92(3):677-94.</p></li>
<li><p>Tang ZH, <strong>Fang Z</strong>, Zeng F, Li Z, Zhou L. Association and interaction analysis of metabolic syndrome and serum uric acid on diastolic heart failure. J Endocrinol Invest. 2013 Sep;36(8):579-83.</p></li>
</ol>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
